Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -38,23 +38,54 @@
             "name": "PostDeploymentAction",
             "displayName": "Post Deployment Action",
             "isExpanded": false,
-            "visibleRule": "WebAppKind != \"\""
+            "visibleRule": "ConnectionType = AzureRM || WebAppKind != \"\""
         },
         {
             "name": "ApplicationAndConfigurationSettings",
             "displayName": "Application and Configuration Settings",
-            "isExpanded": false
+            "isExpanded": false,
+            "visibleRule": "ConnectionType = AzureRM"
         }
     ],
     "inputs": [
+        {
+            "name": "ConnectionType",
```

Comment:
Can we add help text here?
@azooinmyluggage

Reply:
Added through need to check with Atul if changes required in helptext.